modules/ocf_puppet/files/write-otp-config

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ "$#" -ne 2 ]; then
+    echo "Usage: write-otp-config <username> <yubikey public ID>"
+    exit 1
+fi
+
+username="$1"
+yubikeyID="$2"
+
+if grep -q "$username" /opt/puppet/shares/private/otp/yubikeys; then
```

Comment:
grep -q "^$username"

Reply:
actually should be `grep -q '^$username:`